### DISCUSSION TITLE
Nest Hub新功能－睡眠偵測😴

### DISCUSSION THREAD

**Discussion by bonny123865** [1101-GS3073-C/Python-Community-GS3073-C#112] [General]
**新聞大綱:**
Google改進Nest Hub的**睡眠偵測**（Sleep Sensing）功能，現在Nest Hub與舊的模型相比，除了區分睡眠和清醒狀態，甚至進一步能夠偵測使用者整晚快速動眼期、淺層、深層和清醒各種狀態，還可以辨別來自主要用戶的鼾聲。Nest Hub的睡眠偵測功能，使用**微型雷達感測器Soli**，在無接觸和無攝影機的狀況下，掌握用戶睡眠狀態，使得用戶能夠改善睡眠習慣。
還能紀錄用戶發生睡眠中斷的時候，而人們容易忽略造成睡眠中斷的原因，常見的有打鼾和咳嗽。當主要用戶打鼾時，會和Soli的吸氣和呼氣訊號密切對應，當鼾聲來自房間中的其他人，則兩訊號會獨立變化。因此當Nest Hub偵測到音訊和運動特徵不相符的時候，便會將這些聲音標記為其他聲音，紀錄在另一個時間軸上。
現在被應用在追蹤夜間咳嗽的研究上，囊腫性纖維化基金會所資助的一項小型可行性研究，便是嘗試使用Nest Hub來測量，患有罕見疾病**囊腫性纖維**孩童的夜間咳嗽。

**新聞心得:**
在清醒時比較能掌握自己的身體狀況，但在睡覺時比較無意識，而Nest Hub不僅能讓使用者隨時了解自己的睡眠情況、觀察個人睡眠品質之外，還可以偵測夜間睡眠時周遭發出的聲音來源，而除了可以讓大眾正常使用(其實主要是用在影音娛樂方面啦)，也可以用來測量患者狀況，希望它能對於疾病的診斷及治療有很大的幫助！

**新聞來源:**
https://www.ithome.com.tw/news/147763
![0318-nesthub](https://user-images.githubusercontent.com/90883526/141663119-85a73f25-971d-43b9-b0ff-f4fb8f2bd43b.jpg)

**Comment by chia-kai-chang** [comment]
人有三分之一的時間花在睡眠上，如果有產品可以照顧到睡眠的時間，進而提升健康，我認為相當有市場喔。

**Reply by chia-kai-chang** [reply]
文章最後面有提到：

目前睡眠偵測服務已經釋出，在接下來幾周會推送給全球用戶，2022年睡眠偵測會提供免費預覽使用，到了2023年，Google計畫將睡眠偵測功能整合到付費服務Fitbit Premium中。

果然這是一門好生意

**Comment by ghost** [comment]
這個對於長期睡不好的我來說絕對是一門福音阿XDDDD

**Comment by Shiba010** [comment]
睡眠偵測系統對有潛在呼吸中止症的患者非常有幫助！

**Comment by Lynn12378** [comment]
透過睡眠偵測患者疾病去診斷這項功能很有用!
